http-add-on/templates/deployment.yaml

```diff
@@ -3,12 +3,12 @@ kind: Deployment
 metadata:
   labels:
     control-plane: controller-manager
-    httpscaledobjects.http.keda.sh/version: {{ .Values.version }}
+    httpscaledobjects.http.keda.sh/version: {{ .Chart.AppVersion }}
```

Comment:
What if somebody overrides it with ´images.tag´?

(applies to all of these though)